packages/dx-react-grid/docs/guides/editing.md

```diff
@@ -48,10 +48,12 @@ Note, you can also use the `onAddedRowsChange` event to initialize a created row
 
 End-users can click cells and thus activate in-place editors, provided that you include the `TableInlineCellEditing` plugin into the Grid's configuration. To save changes, they can move focus or press Enter. Esc discards changes and deactivates the cell editor.
 
+Note that plugin hasn't built-in tabulation. Users can implement in by define custom `Table.Cell` component. You can see how to override our built-in components in the [Fundamentals Guide](./fundamentals.md#customize-the-appearance).
```

Comment:
`...Users can implement in by define custom...` -> `Users can implement it by defining custom`